src/libraries/System.Diagnostics.Process/src/System/Diagnostics/ProcessManager.Linux.cs

```diff
@@ -4,30 +4,24 @@
 using System.Collections.Generic;
 using System.Globalization;
 using System.IO;
-using System.Linq;
-using System.Text;
 
 namespace System.Diagnostics
 {
     internal static partial class ProcessManager
     {
         /// <summary>Gets the IDs of all processes on the current machine.</summary>
-        public static int[] GetProcessIds()
-        {
-            return EnumerateProcessIds().ToArray();
-        }
+        public static int[] GetProcessIds() => new List<int>(EnumerateProcessIds()).ToArray();
```

Comment:
@stephentoub - is this the "resize at the end" problem you and I discussed the other day?

I'm not sure it is obvious to readers why this approach is preferred over just calling `ToArray()` directly.

Reply:
Using `Enumerable.ToArray` here is fine.  So is `new List<T>(...).ToArray()`.  If there were other reasons to keep a LINQ dependency from System.Diagnostics.Process, we could have left this as is, but as it was the last thing standing in the way of removing that dependency, and switching to use List wasn't problematic, I ditched it.